code/game/objects/items/storage/belt.dm

```diff
@@ -376,6 +376,16 @@
 		/obj/item/restraints/legcuffs/bola,
 		/obj/item/stock_parts/power_store/cell/microfusion, //SKYRAT EDIT ADDITION
 	))
+	// BUBBER EDIT ADDITION BEGIN - Restrict belt items
+	atom_storage.cant_hold = typecacheof(list(
+	/obj/item/gun/ballistic/automatic/rom_flech,
+	/obj/item/gun/syringe,
+	/obj/item/gun/chem,
+	/obj/item/gun/ballistic/automatic/ar,
+	/obj/item/gun/magic/wand,
+	/obj/item/gun/energy/recharge/kinetic_accelerator,
+	/obj/item/gun/energy/plasmacutter))
```

Comment:
Honestly I think we should just get rid of /obj/item/gun in general, this specific blacklist feels like it doesn't handle the issue even that much.

Reply:
Big agree. Officers don't need a reason to hold a bunch of guns on their belt without a specific holster belt or the naked slot

Reply:
> Honestly I think we should just get rid of /obj/item/gun in general, this specific blacklist feels like it doesn't handle the issue even that much.

Well I'm happy to hear! I'll do that, then

Reply:
@Arturlang